### DISCUSSION TITLE
Enhance VertiPaq Analyzer UI with Remaining vpax Data

### DISCUSSION THREAD

**Discussion by corycundy** [TabularEditor/TabularEditor3#1249] [Ideas]
### Is your feature request related to a problem? Please describe

The problem is that the VertiPaq Analyzer collects more data than is displayed within TE3. For example, this includes columns on the Tables tab such as Bid. Filters and MMR.

To see this data, you have to go through the multiple additional steps of exporting the VertiPaq Analyzer results, getting a clean VertiPaq Analyzer Excel sheet, Load the exported results into the spreadsheet and finally view the data there. 

### Describe the solution you'd like

I would like to see the all of the data within the vpax file displayed within the Excel VertiPaq Analyzer to be displayed in the VertiPaq Analyzer within TE3. 

I realize that the many of the most useful data is displayed within TE3, I never understood though in DAX Studio and now TE3, why all of the data wasn't displayed. Everyone may not need all of it, but if it is there, why not just display it. I wouldn't think it would be much of a UI performance hit to load the remaining data. In addition, it could be a toggle in preferences to either show it all or not by default.

The VertiPaq Analyzer spreadsheet has additional tabs for User Hierarchies, Compression, Columns Data Types, Encoding and DAX Expressions. It also has additional columns such as Bid. Filters and MMR in the Tables and Relationships that are missing in TE3. I would think all of it could be loaded relatively easily, similar to how Excel does. 

One may question why even DAX Expressions would be needed. I would argue that if you import a vpax file that someone sent you a while ago, it would be nice to quickly see the DAX expressions. I also realize that a workaround would be to open the vpax file with TE3 vs. import it, but I think it would just be nice to have everything displayed that is already in the vpax file in the same area, similar to the Excel spreadsheet. One could argue that you should have previous versions of DAX code in source control, however, again you'd have to open another tool and one of the strong points of the vpax file is that you may be analyzing a clients model via the vpax file in which you don't have access to their source control.

### Goals

1. Easily see all vpax data after collecting stats or importing a vpax file without having to open another tool. This provides a fluent and productive experience when analyzing a tabular model. The need to go external to see a additional data points in some ways defeats the purpose of having some of the data in the TE3 UI.

### Background

As an alternative to importing all the data, the following columns would be nice to have added in the existing tabs.
Tables Tab: Bid. Filter and MMR
Relationships Tab: Relationship Type, Bid. Filters, MMR

### Additional context

TE3 is an amazing commercial enterprise tool that I spend many hours in. It would be even more amazing to have the comprehensive view as a first class feature without relying on other tools. This would really drive efficiency and productivity when reviewing others models. Maybe DAX Studio would follow suit :)

**Comment by TinyTimothy2** [comment]
and a dream wish, easy export and save and compare process to show changes you have made where you can highlite the improvements.
